zendframework/zendframework/ZF2015-06.yaml

```diff
@@ -0,0 +1,23 @@
+title:     XXE/XEE vector when using ZendXml on multibyte payloads
+link:      http://framework.zend.com/security/advisory/ZF2015-06
+cve:       CVE-2015-5161
+branches:
+    2.0.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.0.0,<2.0.99]
+    2.1.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.1.0,<2.1.99]
+    2.2.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.2.0,<2.2.99]
+    2.3.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.3.0,<2.3.8]
+    2.4.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.4.0,<2.4.6]
+    2.5.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.5.0,<2.5.1]
```

Comment:
I think it's up to each project to determine how they want to advertize vulnerabilities. I agree with @stof, but ultimately, it's @weierophinney call. So, I'm going to merge it as is.